### DISCUSSION TITLE
Can I get free openai API keys for student on education projects purpose?

### DISCUSSION THREAD

**Discussion by naveenalla3000** [community/community#148100] [GitHub Education]
### Select Topic Area

General

### Body

Can I get free openai API keys for student on education projects purpose?

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @Naveenalla3000 , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you to our Education category. Good luck!
